packages/ragbits-document-search/tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+import os
+import shutil
+import tempfile
+from pathlib import Path
+
+import pytest
+
+from ragbits.document_search.documents.sources.base import LOCAL_STORAGE_DIR_ENV
+
+
+@pytest.fixture(scope="module", autouse=True)
+def configure_local_storage_dir():
+    random_tmp_dir = Path(tempfile.mkdtemp())
+    os.environ[LOCAL_STORAGE_DIR_ENV] = random_tmp_dir.as_posix()
+    yield
+    shutil.rmtree(random_tmp_dir)
```

Comment:
Non-blocking:

This should behave the same as the longer version:
```suggestion
def configure_local_storage_dir(tmpdir):
    os.environ[LOCAL_STORAGE_DIR_ENV] = tmpdir
```

Reply:
there is a scope mismatch  for `tmpdir` fixture